media/js/firefox/whatsnew/whatsnew-110-en.js

```diff
@@ -0,0 +1,59 @@
+/*
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ */
+
+function isDefaultBrowser() {
+    'use strict';
+    return new window.Promise(function (resolve, reject) {
+        window.Mozilla.UITour.getConfiguration('appinfo', function (details) {
```

Comment:
Suggestion (non-blocking): this is a nice to have, but should this callback never fire (say if someone has a profile issue so UITour won't work) it would be nice to have some fallback content show instead after X seconds. Could maybe use a timeout here that calls `reject()` unless `cleatTimeout` gets called first?